include/inviwo/core/util/zip.h

```diff
@@ -319,18 +336,23 @@ struct zipIterator {
     }
 
     /**
-     * Should be true if __all__ underlaying iterators are equal.
+     * Should be true if __any__ underlaying iterators are equal.
```

Comment:
Why any, does this make sense for == operator?

Reply:
it's complicated. main reason is that we want to stop as soon as anyone is equal to end...

Reply:
Ok, typo underlaying->underlying

Reply:
@petersteneteg

Reply:
@ResearchDaniel will fix in a different PR